frontend/src/components/ParticipantsList.js

```diff
@@ -0,0 +1,115 @@
+import TableRow from "@material-ui/core/TableRow"
+import Fab from "@material-ui/core/Fab"
+import AddIcon from "@material-ui/icons/Add"
+import { observer } from "mobx-react-lite"
+
+const ParticipantsList = observer(() => {
+  const rootStore = useContext(rootStoreContext)
+  const participantsStore = rootStore.ParticipantStore
+  const [isLoading, setIsLoading] = useState(false)
+
+  // useEffect is a hook that gets called after every render/re-render.  Empty array second argument prevents it from running again.
+  useEffect(() => {
+    setIsLoading(true)
+    participantsStore.getParticipants()
+    setIsLoading(false)
+  }, [])
+
+  return (
+    <Fragment>
+      {isLoading ? (
+        <div>Loading ...</div>
+      ) : (
+        <div>
+          <Breadcrumbs separator="›" aria-label="breadcrumb">
+            <Link color="inherit" href="/">
+              Home
+            </Link>
+            <Typography color="textPrimary">Search Results</Typography>
+          </Breadcrumbs>
+          <Typography variant="h5" color="textPrimary">
+            Participants
+          </Typography>
+          <div className="participants">
+            <Table>
+              <TableHead>
+                <TableRow>
+                  <TableCell>
+                    <Typography>#</Typography>
+                  </TableCell>
+                  <TableCell>
+                    <Typography>PPID</Typography>
+                  </TableCell>
+                  <TableCell>
+                    <Typography>First Name</Typography>
+                  </TableCell>
+                  <TableCell>
+                    <Typography>Last Name</Typography>
+                  </TableCell>
+                  <TableCell>
+                    <Typography>Gender</Typography>
+                  </TableCell>
+                  <TableCell>
+                    <Typography>DOB</Typography>
+                  </TableCell>
+                  <TableCell>
+                    <Typography>Race</Typography>
+                  </TableCell>
+                  <TableCell>
+                    <Typography>Add</Typography>
+                  </TableCell>
+                </TableRow>
+              </TableHead>
+              <TableBody>
+                {participantsStore.participants.map((participant, index) => (
+                  <TableRow key={index}>
+                    <TableCell>
+                      <Typography>Number</Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Typography>{participant.pp_id} </Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Typography>{participant.first_name}</Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Typography>{participant.last_name}</Typography>
+                    </TableCell>
+                    <TableCell>
+                      <Typography>Gender</Typography>
```

Comment:
Why not fill in the other fields: Gender, Race, DOB? They are all available in  the api

Reply:
@hhummel so, this list view isnt really part of this PR.  Its @angelyoung24 work that will be in a separate PR once she has completed her work on her branch.  So its probably not worth commenting on here

Reply:
Got it